build/xbuild.include

```diff
@@ -3,17 +3,6 @@ XBUILD_ARGS=/verbosity:normal /nologo /property:CodePage=65001
 ALL_CSPROJ=$(wildcard *.csproj)
 MAIN_SLN=$(top_builddir)/F-Spot.sln
 
-#if ENABLE_MACPLATFORM
-#PROFILE_NAME=DebugMac
-#endif
-#if ENABLE_GNOMEPLATFORM
-#PROFILE_NAME=Debug
-#endif
-
-#if ENABLE_WINDOWSPLATFORM
-#PROFILE_NAME=DebugWin32
-#endif
-
```

Comment:
Instead of getting rid of this maybe we should just comment it out. I'd actually like to start using this if it makes sense. After we get more of f-spot cleaned up and I want to work on my backend branch and start building in support for mac and windows.